packages/react/src/ReactElement.js

```diff
@@ -12,6 +12,7 @@ import {REACT_ELEMENT_TYPE} from 'shared/ReactSymbols';
 import ReactCurrentOwner from './ReactCurrentOwner';
 
 const hasOwnProperty = Object.prototype.hasOwnProperty;
+const freeze = Object.freeze;
```

Comment:
This is no longer wrapped in __DEV__ - does it still get compiled away in prod? Probably, but idk. Regardless I'd make sure it's a separate commit - doesn't fit with the rest of the changes.